### DISCUSSION TITLE
ChatGLM3-128k 在RAG中的表现远不如CHATGLM3

### DISCUSSION THREAD

**Discussion by 3wweiweiwu** [zai-org/ChatGLM3#1089] [General]
RT
最近在做RAG应用的时候，我对比了CHATGLM3-128K和CHATGLM3原版，我感觉CHATGLM3原版比128K版本的回答准确了很多。在对同样同样大小的文本进行分析的时候， 128K版本完全无法找到答案，而原版却没有任何问题。当我试着对大文本进行问答的时候，128K版本直接无视我的SYSTEM PROMPT，自己开始自问自答。请问这是正常的现象吗？

**Comment by bruleli** [comment]
你好，请问你跑ChatGLM3-6b-128k后，还支持多轮对话吗，我跑了ChatGLM3-6b-128k后，无法进行多轮对话了

**Comment by zRzRzRzRzRzRzR** [comment]
确实有这个问题存在，效果可能不是很尽如人意
